<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PanelContact</class>
 <widget class="QWidget" name="PanelContact">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>136</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PanelContact</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="entireFormVLayout">
     <item>
      <layout class="QHBoxLayout" name="topHLayout">
       <item>
        <widget class="QLabel" name="categoryLabel">
         <property name="text">
          <string>Category: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="categoryLineEdit"/>
       </item>
       <item>
        <spacer name="midHSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="websiteLabel">
         <property name="text">
          <string>Website: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="websiteLineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="midVSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="buttonsLayout">
       <item>
        <spacer name="leftHSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="concatButton">
         <property name="text">
          <string>CONCATENATE</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelButton">
         <property name="text">
          <string>CANCEL</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="rightHSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="displayLabel">
       <property name="text">
        <string>Display...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
